File: web/src/app/admin/models/master-edit-model.ts
import {GeneralSettingVm} from '../../models/general-setting.view.model';

export class MasterEditModel {
  public id: string = '';
  public firstName: string = '';
  public lastName: string = '';
  public middleName: string = '';
  public birthday: string = '';
  public email: string = '';
  public phone: string = '';
  public profiles: GeneralSettingVm[] = [];
}

File: MG.WebAPi/Models/Masters/MasterEditModel.cs
using System;
using System.Collections.Generic;

namespace MG.WebAPi.Models.Masters
{
    public record MasterEditModel
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string MiddleName { get; set; }
        public DateTime? Birthday { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public IEnumerable<GeneralSettingVm> Profiles { get; set; }
    }
}